### PR TITLE
fix: prefix the source secret names with the addon id

### PR DESCRIPTION
**Pull request by yashvardhan-kukreja**
Signed-off-by: Yashvardhan Kukreja <ykukreja@redhat.com>

# py-mtcli

## Description

Short description of the change:

Adds uniqueness to the source/destination secrets map 

## Checklist

**For any modification to `managedtenants/bundles/`**

- [ ] `$ managedtenants --addons-dir=../managed-tenants-bundles/addons --dry-run bundles` (successfuly built all addons)
- [ ] `$ managedtenants -addons-dir=../managed-tenants-bundles/addons --addon-name=<addon> bundles --quay-org <personal_org>` (successfuly built and push a single addon)

### CODE REVIEW

File: managedtenants/utils/ocm.py [hunk 1]
```diff
@@ -335,15 +336,18 @@ def _addon_from_imageset(self, imageset, metadata):
                     continue
                 if key == "config":
                     addon = self.set_addon_config(
-                        addon=addon, config_obj=val, mapped_key=mapped_key
+                        addon=addon,
+                        metadata=metadata,
```
Comment (ashishmax31) [inline managedtenants/utils/ocm.py:340]: Would it be easier to just pass the addonID around? wdyt?

Comment (yashvardhan-kukreja) [reply managedtenants/utils/ocm.py:340]: fair enough, updating :)

**Review by ashishmax31 — APPROVED** [review]

**Pull request merged**